lib/livebook/runtime/evaluator/formatter.ex

```diff
@@ -189,6 +189,9 @@ defmodule Livebook.Runtime.Evaluator.Formatter do
   defp error_context(error) when is_struct(error, Kino.FS.ForbiddenError),
     do: {:file_entry_forbidden, error.name}
 
+  defp error_context(error) when is_struct(error, Mix.Error),
```

Comment:
Mix.Error can happen for several other reasons. Should we check the message as well?

Reply:
Which ones do you suggest? We only show the special action in the setup cell, so I would be fine with not matching the message. Your call!

Reply:
ok, lets give this a try as is then!